### DISCUSSION TITLE
Failed to make bytecode node20-x64 for file

### DISCUSSION THREAD

**Discussion by xorCoding** [yao-pkg/pkg#62] [Q&A] [answered]
Hi, when i use this package to compile my node.js application, i get this warning in the console:
![immagine](https://github.com/yao-pkg/pkg/assets/107063555/ee97d019-badb-4be0-8c6f-ffd43ee241c4)
And if I try to run the executable, i get this error:
![immagine](https://github.com/yao-pkg/pkg/assets/107063555/4d09306a-4b7d-48aa-91f6-02ca0ed58fa6)
This is my server.js file (I removed a lot of stuff to check what could be causing this problem)
![immagine](https://github.com/yao-pkg/pkg/assets/107063555/bfef6a0e-49de-460d-9e11-a5de4d73d551)
I tried both "pkg server.js" and "pkg package.json", but nothing changes.
I have another project where pkg works correctly, what could be the problem here?

**Comment by robertsLando** [comment]
what is your `pkg` config in pakcage.json?

**Reply by xorCoding** [reply]
This is my package.json
![image](https://github.com/yao-pkg/pkg/assets/107063555/a61a2780-5bbf-4c1f-9bb8-f54be080282c)

**Reply by xorCoding** [reply]
I also tried with this configuration:
![image](https://github.com/yao-pkg/pkg/assets/107063555/72d57d07-cc07-4133-b9ca-1e9bb6f2066a)
And the result is the same

**Reply by robertsLando** [reply, marked as answer]
I think the problem is that  `type: "module"`, pkg doesn't work with esm

**Reply by xorCoding** [reply]
Yes, I removed it and now it works. Thanks!
